ONSdigital/eq-terraform
# Split Survey runner infrastructure and application deployment

**What**
Removing the deploy step for the runner application, instead we will use eb deploy going forward.

**How to test**
1. Check out this branch
2. Run  a `terraform apply` in the survey_runner directory
3. Check the application hasn't been deployed
4. From the eq-survey runner repository run
`eb init`
`eb deploy <env.name>`

**Who can review**
Anyone apart from @warrenbailey
